src/link.zig

```diff
@@ -432,6 +435,23 @@ pub const File = struct {
         }
     }
 
+    pub const GotWithAddr = struct { code: []const u8, addr: usize };
+    /// Gets the GOT in it's current form as an allocated string if needed
+    /// Asserts backend uses a got
+    pub fn getGot(base: *File, ally: Allocator) !GotWithAddr {
```

Comment:
Is the `Allocator` really needed here? I think you pass to each backend the allocator instance that they define.